packages/backend/src/models/CatalogModel/CatalogModel.ts

```diff
@@ -46,7 +52,77 @@ export class CatalogModel {
         this.database = args.database;
     }
 
-    // Index catalog happens inside projectModel, inside `saveExploresToCache`
+    async indexCatalog(
```

Comment:
let's bring the comment back? 
```
// Index catalog happens inside projectModel, inside `saveExploresToCache` - and it's a scheduled task
```

Reply:
It no longer happens inside `saveExploresToCache` it happens in `saveExploresToCacheAndIndexCatalog` which is a utility function so we don't repeat code.

On this PR: https://github.com/lightdash/lightdash/pull/12236 we moved it out from `saveExploresToCache` and made it be a scheduled job

Reply:
Ignore actually!

Reply:
ah yes, didn't see your comment before I wrote the one above ^